README.md

```diff
@@ -160,9 +157,9 @@ If you host your express service on a different path than root (/) ie. https://<
 
 ```js
 const Queue = require('bull')
-const { createBullBoard } = require('@bull-board/api')
-const { BullAdapter } = require('@bull-board/api/bullAdapter')
-const { ExpressAdapter } = require('@bull-board/express')
+const {  } = require('@bull-board/api')
```

Comment:
should be removed :]

Reply:
sorry about that 😅 I simultaneously answered your comment on the issue and sent in the PR fixed now